docs/reference/integrations.md

```diff
@@ -2,45 +2,57 @@
 
 ### agent
 
-_Interface_: jenkins_agent_v0
+_Interface_: jenkins_agent_v0    
```

Comment:
There are some trailing whitespaces here

Reply:
Yes, those are intentional so that "Interface" and "Supported charms" would appear on separate lines. Instead of using whitespaces, would it be better practice to separate them with another line like: 

```
5 _Interface_:
6
7 _Supported charms_:
```